src/Templates/parts/base_foot.twig

```diff
@@ -3,5 +3,7 @@
 <script type="text/javascript">
   window.myradio.module = "{{module}}";
   window.myradio.action = "{{action}}";
+  window.memberid = parseInt({{memberid}});
```

Comment:
Maybe keep these inside the window.myradio object (limit things directly in global scope).